README.md

```diff
@@ -114,6 +114,17 @@ There are multiple ways to run this sample: locally using Ollama or Azure OpenAI
 
 See the [cost estimation](./docs/cost.md) details for running this sample on Azure.
 
+#### (Optional) Enable additional user context to Microsoft Defender for Cloud
+In case you have Microsoft Defender for Cloud protection on your Azure OpenAI resource and you want to have additional context on the alerts, run this command:
+```bash
+azd env set MS_DEFENDER_ENABLED true
+```
+
+To customize the application name of the context, run this command:
```

Comment:
need a better sentence
maybe link to the documentation
context is vague

Reply:
will add link to public docs